crates/viewer/re_space_view_spatial/src/visualizers/image_encoded.rs

```diff
@@ -188,19 +187,9 @@ impl ImageEncodedVisualizer {
                 meaning,
                 multiplicative_tint,
                 colormap,
+                "ImageEncoded",
+                &mut self.data,
             ) {
-                // Only update the bounding box if this is a 2D space view.
-                // This is avoids a cyclic relationship where the image plane grows
-                // the bounds which in turn influence the size of the image plane.
-                // See: https://github.com/rerun-io/rerun/issues/3728
-                if spatial_ctx.space_view_class_identifier == SpatialSpaceView2D::identifier() {
-                    self.data.add_bounding_box(
-                        entity_path.hash(),
-                        bounding_box_for_textured_rect(&textured_rect),
-                        spatial_ctx.world_from_entity,
-                    );
-                }
```

Comment:
Is this just no longer necessary?

Reply:
It still is, but moved inside of `textured_rect_from_tensor`